pyca/ui/process_status_collector.py

```diff
@@ -0,0 +1,51 @@
+from prometheus_client.metrics_core import StateSetMetricFamily
+from prometheus_client.registry import REGISTRY
+
+from pyca.db import ServiceStatus, Service
+from pyca.utils import get_service_status
+
+
+class ProcessStatusCollector(object):
+
+    def __init__(self, registry=REGISTRY):
+        registry.register(self)
+
+    def collect(self):
+        service_states = StateSetMetricFamily(
+            'pyca_service_state_info',
+            'Service State of pyCA processes',
+            labels=['service', 'state']
+        )
+
+        service_states.add_metric(
+            ['capture'],
+            self.get_state_dict(ServiceStatus.str(get_service_status(Service.CAPTURE)))
+        )
```

Comment:
You can do somthing like:

```suggestion
        for service in Service.values():
            service_states.add_metric(
                [Service.str(service)],
                self.get_state_dict(get_service_status(service))
            )
```